docs/content/ingestion/tasks.md

```diff
@@ -146,14 +146,18 @@ Append tasks append a list of segments together into a single segment (one after
 
 ### Merge Task
 
-Merge tasks merge a list of segments together. Any common timestamps are merged. The grammar is:
+Merge tasks merge a list of segments together. Any common timestamps are merged.
+If rollup==false, number of rows are not reduced and rows are reordered by timestamp.
```

Comment:
If rollup is disabled as part of ingestion, common timestamps are not merged and rows are reorderded by their timestamp.